src/Io/IniUtil.php

```diff
@@ -10,13 +10,13 @@
     /**
      * Convert a ini like size to a numeric size in bytes.
      *
-     * @param string $iniSetting
+     * @param string $size
      * @return int
      */
     public static function iniSizeToBytes($size)
     {
         if (is_numeric($size)) {
-            return $size;
+            return (int)$size;
```

Comment:
the previous would return a numeric string, not an int

Reply:
Good catch, but see also below for possible overflow :+1: